tedana/workflows/tedana.py

```diff
@@ -162,13 +149,30 @@ def _get_parser():
                         help=argparse.SUPPRESS,
                         action='store_true',
                         default=False)
+    tedpca_args = parser.add_argument_group('Options for controlling TEDPCA')
+    tedpca_args.add_argument('--kdaw',
```

Comment:
Okay, I'll drop these arguments, but will use retain them as arguments to `tedpca` (with the default values) rather than hardcoding them into `kundu_tedpca`. That way, power users can play with them pretty easily when they want to.